waltid-services/waltid-verifier-api/src/main/kotlin/id/walt/verifier/oidc/VerificationUseCase.kt

```diff
@@ -179,12 +180,12 @@ class VerificationUseCase(
     fun getPresentationDefinition(sessionId: String): Result<PresentationDefinition> =
         OIDCVerifierService.getSession(sessionId)?.presentationDefinition?.let {
             Result.success(it)
-        } ?: Result.failure(error("Invalid id provided (expired?): $sessionId"))
+        } ?: Result.failure(IllegalArgumentException("Invalid id provided (expired?): $sessionId"))
```

Comment:
I suggest standardize the message related to the same issue.

Look how many variations there are for the same problem:

"Invalid id provided (expired?): $sessionId"
"No session verification information found for session id!"
"State parameter doesn't refer to an existing session, or session expired"
"State parameter doesn't refer to an existing session, or session expired: $sessionId"